src/url-params.ts

```diff
@@ -27,6 +28,8 @@ export class AnalyticsUrlParams {
 		currentDeviceId: string | null,
 	) {
 		const list = inputIdString ? inputIdString.split(deviceIdSeparator) : [];
+		this.passedDeviceIds = list;
```

Comment:
Better to do a deep copy here Ezequiel. The lines just below might modify `list` and thus `passedDeviceIds` by pushing `currentDeviceId`.

Reply:
Maybe we want to just take the first element `list.length > 0 ? list[0] : null` using the same reasoning of no need to support unknown use cases?

Reply:
You mean something like: 

`const list = inputIdString ? inputIdString.split(deviceIdSeparator) : [];`
`this.passedDeviceIds = inputIdString ? inputIdString.split(deviceIdSeparator) : []; `

?

Reply:
@pranas sorry just saw your comment that was made at the same time. I see what you mean, let me fix that.